src/test/java/Problem2Test.java

```diff
@@ -7,13 +7,23 @@
     public void testRemoveDuplicates() {
         int[][] inputs = {
                 // DO NOT REMOVE OR CHANGE THE GIVEN TEST CASES!
-                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6}
-                // add more test cases
+                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6},
+                {0, 0, 0, 0,},
+                {101, 101, 102, 103, 104, 104, 105},
+                {56, 56, 56, 109, 2039, 2039, 5845, 5845, 5845, 5845},
+                {1, 1, 1, 12, 12, 12, 123, 1234, 1234, 12345},
+                {0, 0, 1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 7, 7, 8, 8, 9, 9},
+                {0, 1, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3}
         };
 
         int[][] expects = {
-                {0, 1, 2, 4, 6}
-                // add more test cases
+                {0, 1, 2, 4, 6},
+                {0},
+                {101, 102, 103, 104, 105},
+                {56, 109, 2039, 5845},
+                {1, 12, 123, 1234, 12345},
+                {0, 1, 2, 3, 4, 5, 6, 7, 8, 9},
+                {0, 1, 2, 3}
```

Comment:
great test. let's add more cases for the edge cases, such as null array and empty array.